modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc

```diff
@@ -4,42 +4,58 @@ include::_attributes.adoc[]
 
-. From the Anypoint Platform Design Center, click *Create*, and then click *Create new application* under *Create a Mule Application*.
-. Enter a project name and click *Create*.
-. Exit the *Let’s Get Started* wizard by clicking *Go straight to canvas*.
-. Click the plus sign.
-. Select a trigger. To have an Amazon S3 event initiate access to your app (that is, to trigger your app), select the Amazon S3 Connector and then select one of the Amazon S3 trigger operations. Alternatively, select an HTTP Listener or Scheduler as the trigger.
+To configure a trigger:
 
+. In Design Center, click *Create*.
+. Click *Create new application*.
+. Specify a value for *Project name*, and click *Create*.
+. Exit from *Let's get started* by clicking *Go straight to canvas*.
+. Click on the Trigger card.
+. If you are using your connector as an input source trigger, search for the connector and select an operation. Otherwise, search for and select *HTTP* or *Scheduler*.
+. Complete the fields with the values you want to configure for your trigger.
++
 You can use the following Amazon S3 operations as triggers:
-
++
 * *On Deleted Object* - Initiates access to your app when an Amazon S3 object is deleted.
 * *On New Object* - Initiates access to your app when an Amazon S3 object is created.
++
+Alternatively, you can specify the trigger as an HTTP Listener as input to your application from a browser or an application like Postman or curl, or you can use Scheduler to provide input to your application at a specific time.
 
 == Configure the Target Component
 
-In Design Center's Flow Designer, after you specify a trigger, you can add the Amazon S3 connector as a component to process the input you received from the trigger. You may also want to add a DataWeave Transform
-between the trigger and the component to transform the input information to what you want as input to the Amazon S3 connector component.
-
-To add an Amazon S3 connector as a component:
-
-. After creating a trigger, click the plus sign.
-. If needed, click *Transform* to use DataWeave to map fields between your received data and what to use with your Amazon S3 component.
-. Select *Amazon S3 Connector*, and then select an operation.
-. Click *Click here to set it up*.
-. Select a connection type and enter the required information:
+. Click “+” next to the trigger card.
+. In *Select a component*, search for and select the connector name.
+. Select an operation for the connector.
+. Enter the required values in the *General* tab.
+. If you are:
++
+** Using the default Amazon S3 storage, leave the default entries for the *Proxy* and the *Advanced* tabs.
+** Connecting to a different storage than the default AWS S3, specify its URL in the the *Advanced* tab's *S3 Compatible Storage URL* field.
++
+. If needed, enter values for other tabs.
+. Specify access information to the connector resource, as described below.
+. If a *Test* button is available, click it to test the connection.
```

Comment:
This generic instruction is not good -- it just sounds like the writer didn't review the UI. The Amazon S3 connector does have a Test button.